10_datastructures_lab/01.py

```diff
@@ -1,10 +1,15 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+import sys
+hosts = {}
+
+#read all hosts from file
+with open('4.Collections\\hosts', 'r') as f:

```

Comment:
1. Better to take the file name from command line
2. Prefer `os.path.join` to concatenate file path parts such as: `os.path.join('4.collections', 'hosts')`